packages/desktop-client/src/components/budget/envelope/budgetsummary/ToBudgetMenu.tsx

```diff
@@ -43,7 +43,7 @@ export function ToBudgetMenu({
       ? [
           {
             name: 'cover',
-            text: t('Cover from a category'),
+            text: t('Cover from category'),
```

Comment:
Could we change the other one to "Cover from a category" instead? Or is there a specific reason to omit 'a' for both?